src/components/ec-summary-info/ec-summary-info.vue

```diff
@@ -0,0 +1,153 @@
+<template>
+  <div
+    class="ec-summary-info"
+    data-test="ec-summary-info"
+  >
+    <div
+      v-if="iconName"
+      class="tw-mr-12"
+    >
+      <ec-icon
+        class="ec-summary-info__main-icon"
+        :data-test="`ec-summary-info__main-icon ec-summary-info__main-icon__${iconName}`"
+        :name="iconName"
+        :size="16"
+      />
+    </div>
+
+    <slot>
+      <div
+        v-if="lineItems.length > 0"
```

Comment:
Hi, 
I see 2 ways to use this component
* with slots, in this case this div will not render anyway even if we do not have this protection
* with passing lines. In this case this protection I feel is no needed because I cannot think of a scenario that we will not pass any lines. 

If we would like to have some kind of protection maybe we can add a prop validator?
Something like that but haven't tested it.
```  lineItems: {
    type: Array,
    default: () => [],
    validator(value) {
      return value.length > 0;
    },
  },

Reply:
the validation would be nice but it would be triggered even when there's a slot. any idea how to solve this?